modules/stochastic_tools/src/utils/AdaptiveMonteCarloUtils.C

```diff
@@ -91,4 +92,40 @@ computeVectorABS(const std::vector<Real> & data)
   return data_abs;
 }
 
+Real
+proposeNewSampleComponentWiseMH(const Real x, const Real rnd1, const Real rnd2)
+{
+  const Real x_new = Normal::quantile(rnd1, x, 1.0);
+  const Real acceptance_ratio = std::log(Normal::pdf(x_new, 0, 1)) - std::log(Normal::pdf(x, 0, 1));
+  const Real new_sample = acceptance_ratio > std::log(rnd2) ? x_new : x;
+  Real val = Normal::cdf(new_sample, 0, 1);
+  return val;
+}
+
+std::vector<Real>
+proposeNewSampleMH(std::vector<const Distribution *> distributions,
+                   const Real rnd,
+                   const std::vector<std::vector<Real>> inputs,
+                   std::vector<std::vector<Real>> inputs_sto)
+{
+  // mooseAssert(distributions.size() == inputs.size() && distributions.size() == inputs_sto.size(),
+  // "Number of inputs inconsistent with number of distributions.");
```

Comment:
Reason why this is commented out?

Reply:
I must have done it accidentally. Just brought it back.